awscli/examples/schemas/create-discoverer.rst

```diff
@@ -0,0 +1,20 @@
+**To create a discoverer**
+
+The following ``create-discoverer`` creates a discoverer. ::
```

Comment:
```suggestion
The following ``create-discoverer`` example creates a discoverer. ::
```

All your examples need to be updated to include the expression:
`The following ``command-name`` example...`
https://aws.github.io/aws-cli/docs_styleguide.html#description

Reply:
updated